src/main/kotlin/game/racingcar/domain/Car.kt

```diff
@@ -2,17 +2,20 @@ package game.racingcar.domain
 
 import game.racingcar.strategy.MoveStrategy
 
-class Car {
-    var blackBox = arrayListOf<Int>()
-    var position: Int = 0
+class Car(val name: String) {
+    private val _blackBox: MutableList<Int> = mutableListOf()
+    private var _position: Int = 0
+
+    val blackBox: List<Int>
+        get() = _blackBox.toList()
+    val position: Int
+        get() = _position
```

Comment:
```suggestion
    private val _blackBox: MutableList<Int> = mutableListOf()

    val blackBox: List<Int>
        get() = _blackBox.toList()
    val position: Int
        get() = _blackBox.lastOrNull()?:0
```
https://github.com/next-step/kotlin-racingcar/pull/572#discussion_r873590876 에 대해서 생각을 해봤는데, move 함수에 대해 _blackBox / _position을 모두 업데이트 시켜주는 것보다, postion의 get()에 처음부터 _blackBox.lastOrNull을 해주는 게 어떨까요? 
코틀린의 위대함입니다! 🤩